src/components/CommentList/CommentList.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Comment } from '../Comment/Comment';
+import { CommentListShape } from '../shapes';
+
+export const CommentList = ({ comments }) => (
+  comments.map(comment => (
+    <Comment key="comment.id" {...comment} />
```

Comment:
```suggestion
    <Comment key={comment.id} {...comment} />
```
it's a string, not an object